carbonblack.json

```diff
@@ -1850,54 +1859,46 @@
                     ]
                 },
                 {
-                    "data_path": "action_result.parameter.requested_status",
+                    "data_path": "action_result.data.*.result",
                     "data_type": "string",
-                    "column_name": "Requested Status",
-                    "column_order": 4,
                     "example_values": [
-                        "Resolved"
+                        "success"
                     ]
                 },
                 {
-                    "data_path": "summary.total_objects",
+                    "data_path": "action_result.summary.Total records updated",
```

Comment:
Should there be underscores instead of spaces for the "Total records updated" object?

Reply:
Done